guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,36 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+If there are any released {ProjectServer} updates, those will be included when performing this update.
```

Comment:
@AkshayGadhaveRH What is the proper way / convention to emphasize this point to users?